src/common/matmul_pd.hpp

```diff
@@ -179,6 +179,14 @@ struct matmul_pd_t : public primitive_desc_t {
         return 1 << (wei_ndims - 2);
     }
 
+    int per_tensor_mask() const {
```

Comment:
Is it possible/reasonable to move this function somewhere like `src/common/utils.h`?

Reply:
OK, got it.